docs/src/sequencepaxos/storage.md

```diff
@@ -126,11 +126,9 @@ my_sled_opts.new(true);
 let my_config = PersistentStorageConfig::with(my_path, my_logopts, my_sled_opts);
 ```
 ## Importing `omnipaxos_storage`
-To use the package, we need to add `omnipaxos_storage` to the dependencies in the cargo file. If a user wish to configure the options for the commit then the user must also add `commitlog` and either `rocksdb` or `sled` as dependencies. Otherwise the default constructor for `PersistentStorageConfig` lets the user create options for the commitlog and the state storage without importing. 
+To use the package, we need to add `omnipaxos_storage` to the dependencies in the cargo file. If a user wishes to configure the options for the commit log, then the user must add `commitlog` as a dependency. For configuring the state storage, either `rocksdb` or `sled` must also be added. Otherwise the default constructor for `PersistentStorageConfig` lets the user create default options for the commit log and the state storage without importing any dependencies.
```

Comment:
It makes more sense to show how to import omnipaxos_storage before showing MemoryStorage and PersistentStorage. So I suggest doing this modification and move it to here: https://github.com/haraldng/omnipaxos/blob/c4d3a83d46f8e78dac4e5cfe93e3eb226307f85e/docs/src/sequencepaxos/storage.md?plain=1#L3

```suggestion
To use the provided storage implementations, we need to add `omnipaxos_storage` to the dependencies in the cargo file.
```